frontend/src/app/app-routing.module.ts

```diff
@@ -1,11 +1,20 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from "@angular/core";
+import { Routes, RouterModule } from "@angular/router";
+import { routes as helloRoutes } from "./hello/routes";
+import { AppComponent } from "./app.component";
 
-
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: "hello",
+    children: [
```

Comment:
You can use `loadChildren: './hello/hello.module.ts#HelloModule',` here for lazy-loading the module routes.

Reply:
O nice, I don't know that as Angular at all ;D